### PR TITLE
Homepage update

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

New homepage for ethereum.org

## Preview Deploy
https://deploy-preview-2124--ethereumorg.netlify.app/en/

### CODE REVIEW

**Comment by samajammin** [comment]
FYI @ryancreatescopy I just force pushed a rebase of latest `dev` - that should fix the build for this PR.

**Comment by samajammin** [comment]
@ryancreatescopy FYI just rebased off `dev` after merging #2260, so preview deploys should now work. Make sure to `git reset --hard origin homepage-updates` before making further changes. You'll also need to [create a personal access token](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) (no scope needed, i.e. no need to check any boxes) & add that to your .env file:
`GITHUB_TOKEN_READ_ONLY_DEV=your_token_here`

**Comment by samajammin** [comment]
@ryancreatescopy this can wait until the PR is finalized but just a reminder to please delete any unused components so it's easier to review the code.

File: src/components/ActionCard.js [hunk 1]
```diff
@@ -22,6 +23,11 @@ const ImageWrapper = styled.div`
   min-height: 260px;
 `
 
+const Title = styled.h3`
+  line-height: 0rem;
```
Comment (samajammin) [inline src/components/ActionCard.js:27]: @ryancreatescopy is this needed? Looks like it's creating issues when the content needs to take up multiple lines, e.g. on https://60087128e474c647250d0c89--ethereumorg.netlify.app/en/contributing/#contributors

Image 2021-01-20 at 10 59 17 AM

File: src/pages/index.js [hunk 2]
```diff
@@ -27,126 +57,880 @@ const Page = styled.div`
...
+  ]
+
+  const features = [
+    {
+      title: price,
+      description: translateMessageId(
+        "page-index-network-stats-eth-price-description",
+        intl
+      ),
+      emoji: ":money_with_wings:",
```
Comment (samajammin) [inline src/pages/index.js:727]: @ryancreatescopy can we delete these emoji fields? Or are we planning to incorporate these into the metric cards somehow? cc @wackerow

**Comment by samajammin** [comment]
FYI @ryancreatescopy @wackerow - I just rebased off `dev` to get the Github API batching . Make sure to `git reset --hard origin/homepage-updates` before making further commits.

**Comment by wackerow** [comment]
@ryancreatescopy I see the issue with the node data that you commented on. 
@samajammin If you run this with `netlify dev` are you getting this? On the current PR deploy preview the etherscan api is returning a `500` from the lambda function, but my attempt (my fork locally/my netlify setup) seems to be working just fine pulling the node count.

**Comment by samajammin** [comment]
FYI @ryancreatescopy @wackerow - I once again rebased off `dev` to fix merge conflicts caused by #2255. Make sure to `git reset --hard origin/homepage-updates` before making further commits.

**Comment by ryancreatescopy** [comment]
@wackerow in the last commit I changed the font sizes on mobile so they're less giant. I can't work out where to change our `<H2>` component however. Mind taking a look? I'd like it to be 24px on small screens.

File: src/theme.js [hunk 3]
```diff
@@ -427,7 +427,7 @@ export const Mixins = {
     font-weight: 400;
   `,
   textLevel2: `
-    font-size: 2rem;
+    font-size: 1.5rem;
```
Comment (ryancreatescopy) [inline src/theme.js:430]: It looks like this change has affected H2 size on regular desktop too.
image

To clarify:
- on M and above, H2's should be 2rem (32px)
- below M, H2s should be 1.5rem (24px)

**Review by ryancreatescopy — COMMENTED** [review]
All codeblocks are wrapping nicely other than the javascript example

**Review by samajammin — APPROVED** [review]

**Pull request merged**